common/include/pcl/PCLHeader.h

```diff
@@ -14,25 +15,22 @@ namespace pcl
 {
   struct PCLHeader
   {
-    PCLHeader (): seq (0), stamp ()
-    {}
-
     /** \brief Sequence number */
-    pcl::uint32_t seq;
+    pcl::uint32_t seq = 0;
     /** \brief A timestamp associated with the time when the data was acquired
       *
       * The value represents microseconds since 1970-01-01 00:00:00 (the UNIX epoch).
       */
-    pcl::uint64_t stamp;
+    pcl::uint64_t stamp = 0;
     /** \brief Coordinate frame ID */
     std::string frame_id;
 
-    using Ptr = boost::shared_ptr<PCLHeader>;
-    using ConstPtr = boost::shared_ptr<const PCLHeader>;
+    using Ptr = std::shared_ptr<PCLHeader>;
```

Comment:
Pinging @taketwo regarding the changes to smart pointers from boost to std

Reply:
Yes, please leave this out for now. The transition from `boost` to `std` smart pointers will happen all at once in the entire library.